src/pages/UserSettings/UserSettings.tsx

```diff
@@ -181,7 +184,7 @@ export default function UserSettings() {
                 setEmailTimeNetworkingState(Constants.LOADING_MUTATION_TEXT);
                 setEmailTime(emailTime);
               }}
-              label="Pick a time to receive notifications:"
+              label="Reminder Time"
               type="time"
             />
             <label className="font-normal text-gray-600">
```

Comment:
just curious why are you using a separate lab instead of the TextField's supportingText/errorText props?

Reply:
I swear at one point the design showed a separate text but I guess it's gone now 🥲 so I'll update all of these now...